src/ring_theory/dedekind_domain/ideal.lean

```diff
@@ -858,6 +859,64 @@ end is_dedekind_domain
 
 end height_one_spectrum
 
+section
+
+open ideal
+
+variables {R} {A} [is_domain R] [is_dedekind_domain A] {I : ideal R} {J : ideal A}
+
+/-- The map from ideals of `R` dividing `I` to the ideals of `A` dividing `J` induced by
+  a homomorphism `f : R/I →+* A/J` -/
+@[simps]
+def ideal_factors_fun_of_quot_hom {f : R ⧸ I →+* A ⧸ J} (hf : function.surjective f ) :
+  {p : ideal R | p ∣ I} →o {p : ideal A | p ∣ J} :=
```

Comment:
Can you add the lemma that says `ideal_factors_fun_of_quot_hom (ring_hom.id (A ⧸ J)) = order_hom.id` or whatever the correct spelling is?